<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<include file="$(find quicksilver_description)/launch/urdf_gazebo_view.launch" />
		


	<rosparam file="$(find quicksilver_description)/config/joint_velocity.yaml" command="load"/> 

	
	<node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/open_base" 
	args="--namespace=/open_base
	joint_state_controller
	left_joint_velocity_controller
	back_joint_velocity_controller
	right_joint_velocity_controller
	front_joint_velocity_controller
	"/>
	<param name="/joy_node/dev" type="str" value="/dev/input/js1" />
	<node name="joy_node" pkg="joy" type="joy_node"/>
	<node name="quicksilver_node" pkg="quicksilver_description" type="joy_to_quicksilver.py" />


</launch>
